tests/vcd_vapp.py

```diff
@@ -150,15 +151,15 @@ def test_111_instantiate_vapp_custom_disk_size(self):
         assert len(vms) > 0
 
         items = vms[0].xpath('//ovf:VirtualHardwareSection/ovf:Item',
-            namespaces={'ovf': NSMAP['ovf']})
+                             namespaces={'ovf': NSMAP['ovf']})
```

Comment:
Don't forget to run flake8 to verify that these extra indentations are acceptable

Reply:
done.